shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -96,29 +150,25 @@ - (void) setupGo
   BOOL securityAccessGroupOverride = isSimulator;
   BOOL skipLogFile = false;
 
-  NSString * home = NSHomeDirectory();
-
-  NSString * keybasePath = [@"~/Library/Application Support/Keybase" stringByExpandingTildeInPath];
-  NSString * levelDBPath = [@"~/Library/Application Support/Keybase/keybase.leveldb" stringByExpandingTildeInPath];
-  NSString * chatLevelDBPath = [@"~/Library/Application Support/Keybase/keybase.chat.leveldb" stringByExpandingTildeInPath];
-  NSString * eraseableKVPath = [@"~/Library/Application Support/Keybase/eraseablekvstore/device-eks" stringByExpandingTildeInPath];
-  NSString * logPath = [@"~/Library/Caches/Keybase" stringByExpandingTildeInPath];
-  NSString * serviceLogFile = skipLogFile ? @"" : [logPath stringByAppendingString:@"/ios.log"];
-
-  // Make keybasePath if it doesn't exist
-  [self createBackgroundReadableDirectory:keybasePath setAllFiles:YES];
-  [self addSkipBackupAttributeToItemAtPath:keybasePath];
-
+  NSString* home = NSHomeDirectory();
+  NSString* sharedHome = [[[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:@"group.keybase"] relativePath];
+  sharedHome = [self setupHome:home sharedHome:sharedHome];
```

Comment:
should we log some error or warning if `home == sharedHome`? we don't really expect it to happen right?

Reply:
Shouldn't happen.